index.html

```diff
@@ -353,6 +353,60 @@
   <section id="conformance">
     <p>There is only one class of product that can claim conformance to this specification: a <dfn>user agent</dfn>.</p>
   </section>
+  
+  <section>
+    <h2>IANA Considerations</h2>
+    <p>The link relation types below have been registered by IANA per Section 6.2.1 of [[!RFC5988]]:</p>
+    
+    <section>
+      <h3>dns-prefetch Link Relation Type</h3>
+      <dl>
+        <dt>Relation Name:</dt>
+        <dd>dns-prefetch</dd>
+        <dt>Description:</dt>
+        <dd>The dns-prefetch link relation type is used to indicate an origin that will be used to fetch required resources, and that the user agent SHOULD resolve as early as possible.</dd>
```

Comment:
On 2016-01-24 10:23, Ilya Grigorik wrote:
> <https://github.com/w3c/resource-hints/pull/53#discussion_r50639577>:
> Hmm, actually...
> https://www.w3.org/respec/guide.html#inclusions-transformations

that's just about external resources and how to include them, if i read 
it correctly.

are you suggesting we should treat those descriptions likes that? have 
an external resource for each description, and then include them at the 
two locations where they are needed? that would probably work, but seems 
like a rather awful hack around the lack of a macro facility.